src/custom/utils/metadata.ts

```diff
@@ -23,6 +28,11 @@ export type AppDataDoc = {
 
 export const DEFAULT_APP_CODE = 'CowSwap'
 
+const pinata = pinataSDK(
+  process.env.REACT_APP_PINATA_API_KEY as string,
+  process.env.REACT_APP_PINATA_SECRET_API_KEY as string
```

Comment:
With the `as` u are saying that these params are provided always? 
Shouldn't you have a conditional logic? even if it's just for throwing an error if they are not provided.